### PR TITLE
add comment about edged.podsandbox-image

### LINKED ISSUE #1093: failed to read pod IP on RPi 4
**What happened**:

1. run cloudcore on cloud side (centos vm) 
2. run edgecore on edge side (RPi 4)
3. run scale command on cloudside
```
kubectl scale deploy arm --replicas=1
```
4. get error info from edgecore
```
W0904 13:50:04.238289    1860 docker_sandbox.go:384] failed to read pod IP from plugin/docker: Couldn't find network status for default/arm-649957569d-8vrw8 through plugin: invalid network status for
W0904 13:50:04.263502    1860 docker_sandbox.go:384] failed to read pod IP from plugin/docker: Couldn't find network status for default/arm-649957569d-8vrw8 through plugin: invalid network status for
I0904 13:50:05.261254    1860 generic.go:81] GenericLifecycle: Relisting
W0904 13:50:05.272757    1860 docker_sandbox.go:384] failed to read pod IP from plugin/docker: Couldn't find network status for default/arm-649957569d-8vrw8 through plugin: invalid network status for
I0904 13:50:06.293981    1860 generic.go:81] GenericLifecycle: Relisting
W0904 13:50:06.306901    1860 docker_sandbox.go:384] failed to read pod IP from plugin/docker: Couldn't find network status for default/arm-649957569d-8vrw8 through plugin: invalid network status for
I0904 13:50:07.332697    1860 generic.go:81] GenericLifecycle: Relisting
W0904 13:50:07.343673    1860 docker_sandbox.go:384] failed to read pod IP from plugin/docker: Couldn't find network status for default/arm-649957569d-8vrw8 through plugin: invalid network status for
I0904 13:50:08.365351    1860 generic.go:81] GenericLifecycle: Relisting
W0904 13:50:08.378234    1860 docker_sandbox.go:384] failed to read pod IP from plugin/docker: Couldn't find network status for default/arm-649957569d-8vrw8 through plugin: invalid network status for

```

other information:

deploy arm information:
```
kubectl get deploy arm -o yaml
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  annotations:
    deployment.kubernetes.io/revision: "2"
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"apps/v1","kind":"Deployment","metadata":{"annotations":{},"labels":{"app":"arm"},"name":"arm","namespace":"default"},"spec":{"replicas":1,"selector":{"matchLabels":{"app":"arm"}},"template":{"metadata":{"labels":{"app":"arm"}},"spec":{"containers":[{"image":"arm32v7/nginx","name":"nginx","ports":[{"containerPort":80,"hostPort":8091}]}],"nodeSelector":{"arch":"armv7l","name":"edge-node"}}}}}
  creationTimestamp: "2019-08-29T01:21:22Z"
  generation: 22
  labels:
    app: arm
  name: arm
  namespace: default
  resourceVersion: "1359903"
  selfLink: /apis/extensions/v1beta1/namespaces/default/deployments/arm
  uid: 40e2ea5a-8740-44f3-a5a4-8022c4b47f4c
spec:
  progressDeadlineSeconds: 600
  replicas: 1
  revisionHistoryLimit: 10
  selector:
    matchLabels:
      app: arm
  strategy:
    rollingUpdate:
      maxSurge: 25%
      maxUnavailable: 25%
    type: RollingUpdate
  template:
    metadata:
      creationTimestamp: null
      labels:
        app: arm
    spec:
      containers:
      - image: arm32v7/nginx
        imagePullPolicy: Always
        name: nginx
        ports:
        - containerPort: 80
          hostPort: 8091
          protocol: TCP
        resources: {}
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
      dnsPolicy: ClusterFirst
      nodeSelector:
        arch: armv7l
        name: edge-node
      restartPolicy: Always
      schedulerName: default-scheduler
      securityContext: {}
      terminationGracePeriodSeconds: 30
status:
  conditions:
  - lastTransitionTime: "2019-08-29T03:26:13Z"
    lastUpdateTime: "2019-08-29T03:26:13Z"
    message: ReplicaSet "arm-649957569d" has successfully progressed.
    reason: NewReplicaSetAvailable
    status: "True"
    type: Progressing
  - lastTransitionTime: "2019-09-04T05:49:55Z"
    lastUpdateTime: "2019-09-04T05:49:55Z"
    message: Deployment does not have minimum availability.
    reason: MinimumReplicasUnavailable
    status: "False"
    type: Available
  observedGeneration: 22
  replicas: 1
  unavailableReplicas: 1
  updatedReplicas: 1

```

edge node information:
```
 kubectl get node 192.168.4.27-pi1 -o yaml
apiVersion: v1
kind: Node
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"v1","kind":"Node","metadata":{"annotations":{},"labels":{"arch":"armv7l","name":"edge-node","node-role.kubernetes.io/edge":""},"name":"192.168.4.27-pi1"}}
    node.alpha.kubernetes.io/ttl: "0"
  creationTimestamp: "2019-08-29T06:20:22Z"
  labels:
    arch: armv7l
    name: edge-node
    node-role.kubernetes.io/edge: ""
  name: 192.168.4.27-pi1
  resourceVersion: "1360409"
  selfLink: /api/v1/nodes/192.168.4.27-pi1
  uid: cb643fab-95b0-4d3e-81e1-26d7c1c37fd7
spec:
  podCIDR: 10.244.5.0/24
status:
  addresses:
  - address: 192.168.4.27
    type: InternalIP
  - address: 192.168.4.27-pi1
    type: Hostname
  allocatable:
    cpu: "4"
    memory: 1839Mi
    pods: "110"
  capacity:
    cpu: "4"
    memory: 1939Mi
    pods: "110"
  conditions:
  - lastHeartbeatTime: "2019-09-04T05:56:17Z"
    lastTransitionTime: "2019-09-04T05:56:17Z"
    message: edge is posting ready status
    reason: EdgeReady
    status: "True"
    type: Ready
  daemonEndpoints:
    kubeletEndpoint:
      Port: 0
  nodeInfo:
    architecture: arm
    bootID: ""
    containerRuntimeVersion: remote://19.3.1
    kernelVersion: 4.19.57-v7l+
    kubeProxyVersion: ""
    kubeletVersion: v1.15.3-kubeedge-v1.0.0-347+9a7c759a7440f9
    machineID: ""
    operatingSystem: linux
    osImage: Raspbian GNU/Linux 10 (buster)
    systemUUID: ""
  phase: Running

```

pod information:
```
kubectl get pod
NAME                   READY   STATUS               RESTARTS   AGE
arm-649957569d-8vrw8   0/1     ContainerCannotRun   2          6m55s

```

describe pod information:
```
kubectl describe pod arm-649957569d-8vrw8
Name:           arm-649957569d-8vrw8
Namespace:      default
Priority:       0
Node:           192.168.4.27-pi1/192.168.4.27
Start Time:     Wed, 04 Sep 2019 13:49:56 +0800
Labels:         app=arm
                pod-template-hash=649957569d
Annotations:    <none>
Status:         Running
IP:
Controlled By:  ReplicaSet/arm-649957569d
Containers:
  nginx:
    Container ID:   docker://1f45813031be889316ae1f2b0c7038191866f4b1393eec7d4c35022c8d22973e
    Image:          arm32v7/nginx
    Image ID:       docker-pullable://arm32v7/nginx@sha256:e5e4a3e39e58dc4c40064c9433740e7c2031516375c201aab26649dcfa2c7a6d
    Port:           80/TCP
    Host Port:      8091/TCP
    State:          Terminated
      Reason:       ContainerCannotRun
      Message:      cannot join network of a non running container: 01f739b86e0ab741f481225452dcdafe0ba8d7abb46bdaefe2d91c00e11ef5e4
      Exit Code:    128
      Started:      Wed, 04 Sep 2019 13:52:13 +0800
      Finished:     Wed, 04 Sep 2019 13:52:13 +0800
    Last State:     Terminated
      Reason:       ContainerCannotRun
      Message:      cannot join network of a non running container: 01f739b86e0ab741f481225452dcdafe0ba8d7abb46bdaefe2d91c00e11ef5e4
      Exit Code:    128
      Started:      Wed, 04 Sep 2019 13:52:13 +0800
      Finished:     Wed, 04 Sep 2019 13:52:13 +0800
    Ready:          False
    Restart Count:  2
    Environment:    <none>
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-dv49d (ro)
Conditions:
  Type           Status
  Initialized    True
  Ready          False
  PodScheduled   True
Volumes:
  default-token-dv49d:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  default-token-dv49d
    Optional:    false
QoS Class:       BestEffort
Node-Selectors:  arch=armv7l
                 name=edge-node
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type    Reason     Age    From               Message
  ----    ------     ----   ----               -------
  Normal  Scheduled  7m23s  default-scheduler  Successfully assigned default/arm-649957569d-8vrw8 to 192.168.4.27-pi1

```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeEdge version: v1.0.0-347+9a7c759a7440f9
- Hardware configuration:
edged: RPi 4

- OS (e.g. from /etc/os-release): 
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
- Kernel (e.g. `uname -a`): 
Linux 192.168.4.27-pi1 4.19.57-v7l+ #1244 SMP Thu Jul 4 18:48:07 BST 2019 armv7l GNU/Linux
- Others:

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: zhangjie <iamkadisi@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1093 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/conf/edge.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ edged:
     remote-runtime-endpoint: unix:///var/run/dockershim.sock
     remote-image-endpoint: unix:///var/run/dockershim.sock
     runtime-request-timeout: 2
-    podsandbox-image: k8s.gcr.io/pause
+    podsandbox-image: kubeedge/pause:3.1 # kubeedge/pause:3.1 for x86 arch , kubeedge/pause-arm:3.1 for arm arch, kubeedge/pause-arm64 for arm64 arch
```
Comment (kadisi) [inline edge/conf/edge.yaml]: now kubeedge/pause:3.1 not exist. we need to push `pause:3.1` to kubeedge
@kevin-wangzefeng @fisherxu 

image

Comment (fisherxu) [reply edge/conf/edge.yaml]: Thanks @kadisi , I have pushed the `pause:3.1` image.

**Pull request merged**